assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceErrorModal.js

```diff
@@ -181,7 +181,10 @@ export default function AudienceErrorModal( {
 
 					trackEvent( trackEventCategory, action );
 				} }
-				onClose={ onCancel }
+				onClose={ () => {
+					onCancel();
+					handleDialog();
+				} }
```

Comment:
Is this actually required? The component isn't mentioned in the AC or IB, and has an `onCancel()` handler...

If it _is_ needed, please mention it in the QAB otherwise it won't get tested.

Reply:
I don't think its required, it was added when I was searching for components using `ModalDialog` component and there is was added. Removed it now.